### ISSUE TITLE
Radio button grouping

### ISSUE THREAD

**Issue by Windvis** [appuniversum/ember-appuniversum#210]
I've seen some projects use a layout similar to something like this for a group of radio buttons:

```hbs
<AuLabel for="radios">Some label that describes the radio options</AuLabel>
<ul id="radios">
  <li><AuControlRadio @label="option 1" /></li>
  <li><AuControlRadio @label="option 2" /></li>
...
</ul>
```

I think the more semantically correct version would use a [`fieldset`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) and `legend` combination instead?

```hbs
<fieldset>
  <legend>Some label that describes the radio options</legend>
  <AuControlRadio @label="option 1" />
  <AuControlRadio @label="option 2" />
...
</fieldset>
```

At the moment appuniversum doesn't ship any styles for this setup I think. Would it make sense to add some? 

Or a dedicated component that could provide some abstraction for it?

```hbs
<AuRadioGroup @error={{this.hasError}} @required={{this.isRequired}} @disabled={{this.isDisabled}}>
  <:label>Some label that describes the radio options</label>
  <:radios as |Radio|>
    <Radio @label="option 1" />
    <Radio @label="option 2" />
  </radios>
</AuRadioGroup>

**Comment by Windvis** [comment]
A more simple alternative could be to allow changing the rendered element for the `AuLabel` component, so that it can also be used for the legend of a fieldset. The wrapper component doesn't provide a huge benefit over the native fieldset, so maybe just documenting that layout is better?

**Comment by Dietr** [comment]
Adding a legend and fieldset component is an option. The only reason is so we can provide a consistent style without having to add classes on it.

I think it's best to avoid making components like AuLabel behave like a legend. Some components already have that mistake.

**Comment by Windvis** [comment]
Fixed by #225
